admin/client/src/components/form-action/styles.scss

```diff
@@ -95,6 +95,10 @@
 	&:focus {
 		background-color: $gray-lighter;
 	}
+
+	.toolbar--content & {
```

Comment:
That's technically not BEM, but i'll let it slide for now because we need some class refactor in there anyway